java/ingest/ingest-status-store/src/test/java/sleeper/ingest/status/store/job/QueryIngestJobStatusByPeriodIT.java

```diff
@@ -86,18 +86,16 @@ public void shouldExcludeIngestJobInOtherTable() {
     }
 
     @Test
-    public void shouldIncludeFinishedStatusUpdateOutsidePeriod() throws Exception {
+    public void shouldIncludeFinishedStatusUpdateOutsidePeriod() {
         // Given
         IngestJob job = jobWithFiles("file");
-        Instant periodStart = Instant.now().minus(Period.ofDays(1));
-        Instant startedTime = Instant.now();
+        Instant startedTime = Instant.parse("2023-01-03T14:50:00.001Z");
+        Instant periodStart = startedTime.minus(Period.ofDays(1));
 
         // When
         store.jobStarted(DEFAULT_TASK_ID, job, startedTime);
-        Thread.sleep(1);
-        Instant periodEnd = Instant.now();
-        Thread.sleep(1);
-        Instant finishedTime = Instant.now();
+        Instant finishedTime = Instant.parse("2023-01-03T14:56:00.001Z");
+        Instant periodEnd = finishedTime.plus(Period.ofDays(1));
```

Comment:
This is incorrect, it doesn't match what the test was doing before. The order the times happen should be periodStart, startedTime, periodEnd, finishedTime.